core/Http.php

```diff
@@ -22,7 +22,7 @@ class Http
     /**
      * Returns the "best" available transport method for {@link sendHttpRequest()} calls.
      *
-     * @return string Either `'curl'`, `'fopen'` or `'socket'`.
+     * @return string|null Either curl, fopen, socket or null of no method is supported.
```

Comment:
I can revert, I just found it a bit extreme to have those keywords quoted in both simple quotes and then backticks (i'm not against formatting but here I thought it wasn't helping readability, rather the opposite)